src/styles/accessibility.scss

```diff
@@ -35,3 +37,6 @@
 .linkUnderline {
   text-decoration: underline;
 }
+
+$focus-box-shadow: 0 0 1px 5px var(--highlight-color, variables.$white),
+  0 0 12px 6px rgb(0 0 0 / 80%);
```

Comment:
Are you still using this?

Reply:
Yes we are, in the mixin at the top of the file:
`@mixin focus-visible($value: $focus-box-shadow, $default-value: none)`

I will move this focus-box-shadow above said mixin to prevent any confusion!